developer_docs/tools/playwright-mcp-guide.md

```
@@ -0,0 +1,205 @@
+
+### Stale refs
+
+Snapshot refs (`eXXX`) **change after every page interaction**. If you get a "ref not found"
+error, re-run `browser_snapshot` to get fresh refs before retrying.
+
+---
+
+## Clicking Patterns
+
+### Standard click
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)